pallets/nft/src/benchmarking.rs

```diff
@@ -26,164 +27,75 @@ fn dollar(d: u32) -> u128 {
 	d.saturating_mul(100_000_000_000_000)
 }
 
+fn do_create_class<T: Config>() -> (T::AccountId, <T::Lookup as StaticLookup>::Source, BoundedVecOfUnq<T>) {
```

Comment:
I think it would be good to comment the functions with reasoning why we think it's the worst case scenario in bench files. This is not related to just this case but to all cases. Creating issue for this.

Reply:
this should be in changed in nft PR @green-jay